src/refactor/index.jsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+
+
+// return on each Click the value of Square
+function Square(props) {
+  return (
+    <button className="square" onClick={props.onClick}>
+      {props.value}
+    </button>
+  )
+}
+// Describe Board Game + set boolean to setup user 
+class Board extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      squares: Array(9).fill(null),
+      xIsNext: true,
+    };
+  }
+
+  // handle which turn is it
+  handleClick(i) {
+    const squares = this.state.squares.slice();
+    squares[i] = this.state.xIsNext ? 'X': 'O';
+    this.setState({
+      squares: squares,
+      xIsNext : !this.state.xIsNext,
+    });
+    
+}
+
+
+  renderSquare(i) {
+    return (
+      <Square
+        value={this.state.squares[i]}
+        onClick={() => this.handleClick(i)}
+      />
+    );
+    
+   
+  }
+  
+  render() {
+    
+    const winner = calculateWinner(this.state.squares);
+    let status;
+    if (winner) {
+      status = 'Winner:' + winner
+    } else {
+      status = 'Next player:' + (this.state.xIsNext ? 'X' : 'O');
+    }
+
+    return (
+      <div>
+        <div className="status">{status}</div>
+        <div className="board-row">
+          {this.renderSquare(0)}{this.renderSquare(1)}{this.renderSquare(2)}
+        </div>
+        <div classNagitme="board-row">
+          {this.renderSquare(3)}{this.renderSquare(4)}{this.renderSquare(5)}
+        </div>
+        <div className="board-row">
+          {this.renderSquare(6)}{this.renderSquare(7)}{this.renderSquare(8)}
+        </div>
+      </div>
+    );
+  }
+}
+
+class Game extends React.Component {
+  render() {
+    return (
+      <div className="game">
+        <div className="game-board">
```

Comment:
This div belongs to the Board component itself.

Reply:
Why?
when I try to move it to the Board component it's messed up. 
it works that way, what difference does it make?

Reply:
One of the biggest components strength is being reusable the way we accomplish it is by putting all the necessaries inside the same scope, let say you want to reuse the <Board/> you must wrap it with another div every time you will want to use it otherwise the board will be broken.

Scoping together relevant code is an important concept in development for many good reasons and not only because of reusability but also because of maintainability and etc.